query.go

```diff
@@ -236,7 +241,7 @@ func (r *dhtQueryRunner) queryPeer(proc process.Process, p peer.ID) {
 
 	// make sure we're connected to the peer.
 	// FIXME abstract away into the network layer
-	if conns := r.query.dht.host.Network().ConnsToPeer(p); len(conns) == 0 {
+	if r.query.dht.host.Network().Connectedness(p) == inet.NotConnected {
```

Comment:
maybe add a comment to the effect of "failure to establish a connection will cause the function to short circuit in this block" to make the function flow read easier

Reply:
👌 done